### ISSUE TITLE
Global indent width setting

### ISSUE THREAD

**Issue by ipeglin** [astral-sh/ruff#11228]
I would very much like to see a global VSCode extension setting for indent width, that omits the need for a `ruff.toml` or `pyproject.toml`. I personally prefer a different indent size than 4, and it seems really unnecessary to constantly create a local configuration file for such a small change in the config.

**Comment by MichaReiser** [comment]
You can change the `formatArgs` and `lintArgs` to include a `--config indent-width=2`. I'm not sure if the new LSP will have a setting for this that doesn't require a configuration file. CC: @snowsignal

**Comment by charliermarsh** [comment]
Yeah this is currently solvable without creating a configuration file, with the following in your `settings.json`:

```json
{
    "ruff.lint.args": ["--config", "indent-width=2"],
    "ruff.format.args": ["--config", "indent-width=2"],
}
```

**Comment by snowsignal** [comment]
Even though the current LSP supports this, our new LSP would require this to be a configuration file at the moment, and I'd like to have this as an proper editor setting.
